### ISSUE TITLE
Connecting to Gnome remote desktop gets wrong desktop geometry

### ISSUE THREAD

**Issue by jpmorrison** [ultravnc/UltraVNC#29]
UltraVNC 1.3,4,2 (Windows 10)

Connecting to Fedora 34 - Gnome remote desktop VNC enabled in settings gets the wrong desktop size so the display is cut off.  Workaround is to set to half-size in UltraVNC or change the display in Fedora. It's using 800x600 when the real display is 1366x768

![image](https://user-images.githubusercontent.com/1448696/139559861-d248bb32-1b91-4a4b-a2ae-597c38337cef.png)

- Maybe a Gnome issue? Not sure how to check
- I wasn't able to connect at first to Fedora, TLS-anon is the default. 'none' needs to be enabled in gsettings/dconf-editor
- If you don't see 'none' in encryption you won't be able to connect - I don't think UltraVNC has TLS

```
gsettings  list-recursively  org.gnome.desktop.remote-desktop.vnc
org.gnome.desktop.remote-desktop.vnc view-only false
org.gnome.desktop.remote-desktop.vnc auth-method 'password'
org.gnome.desktop.remote-desktop.vnc encryption ['none', 'tls-anon']
```

**Comment by Neustradamus** [comment]
@RudiDeVos: Please read here.

**Comment by RudiDeVos** [comment]
Does someone know were a virtual image of Fedora with remote desktop can be downloaded ?
Always spending a lot of time on installing and configuring OS's before i even can start testing...

**Comment by juanpc2018** [comment]
In Kubuntu / Ubuntu / pearOS any KDE environment,
the best are KRFB "Server" & KRDC "Viewer"

x11vnc has geometry problems..

**Comment by RudiDeVos** [comment]
Fixed in 1.3.8.0
https://uvnc.com/downloads/ultravnc/149-ultravnc-1-3-8-0.html
